### DISCUSSION TITLE
Gesture typing? Voice typing?

### DISCUSSION THREAD

**Discussion by rafnov** [FossifyOrg/General-Discussion#628] [Ideas]
Are there any plans for gesture / swipe typing?
I miss that very much.
Also when I try to enable voice typing it says: no proper application found. What do I miss here?

**Comment by naveensingh** [comment]
> Gesture typing?

Please raise a [feature request](https://github.com/FossifyOrg/Keyboard/issues/new?assignees=&labels=feature+request%2Cneeds+triage&projects=&template=feature_request.yml
) for this.

> no proper application found

You need to install a Voice Input Method, Fossify Keyboard by itself doesn't have capability to transcribe speech.

**Reply by rafnov** [reply]
> You need to install a Voice Input Method, Fossify Keyboard by itself doesn't have capability to transcribe speech.

What is that? No package with such name, but I have Google "voice typing" (direct translation from polish) installed.
FK can't use it?

**Reply by naveensingh** [reply]
Please check if "Google Voice Typing" is enabled in `System settings => System => Languages & Input => On screen keyboard`.

**Reply by rafnov** [reply]
No such setting on a testing device with Android 10, but voice typing works when Gboard is enabled.
